framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/CopyText/index.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import * as React from 'react';
+import styles from './CopyText.module.scss';
+import Icon from '../Icon';
+import Text from '../Text';
+
+interface Props {
+	name: string;
+	size?: 's' | 'm' | 'l' | 'xl';
+	color?: 'green' | 'pink' | 'yellow' | 'blue' | 'white' | 'gray' | 'red';
+	type?: 'h1' | 'h2' | 'h3' | 'th' | 'tr' | 'p';
+}
+
+const CopyText: React.FC<Props> = (props: Props) => {
+	const [hover, setHover] = React.useState(true);
+	const [name, setName] = React.useState(props.name);
+
+	const clipToClipboard = async (text: string) => {
+		setHover(false);
+		setName('Copied');
+		setTimeout(() => {
```

Comment:
i think the timeout id need to be cleaned up in case of the component being removed?